src/trove_classifiers/__init__.py

```diff
@@ -59,6 +59,7 @@
     "Environment :: GPU :: NVIDIA CUDA :: 12 :: 12.4",
     "Environment :: GPU :: NVIDIA CUDA :: 12 :: 12.5",
     "Environment :: GPU :: NVIDIA CUDA :: 12 :: 12.6",
+    "Environment :: GPU :: NVIDIA CUDA :: 13",
```

Comment:
Should there be a `Environment :: GPU :: NVIDIA CUDA :: 13 :: 13.0` added as well to match the other ones?